tests/helpers/perf_run_helper.rb

```diff
@@ -549,9 +522,9 @@ def copy_scale_results(scenario)
 
     # copy puppet-metrics-collector to scale results dir (this iteration) and parent dir (entire scale run)
     # TODO: rename dir to 'puppet-metrics-collector'
-    src = "#{@archive_root}/puppet_metrics_collector"
-    FileUtils.copy_entry src, "#{scale_result_dir}/puppet_metrics_collector"
-    FileUtils.copy_entry src, "#{scale_results_parent_dir}/puppet_metrics_collector"
+    src = "#{@archive_root}/#{PUPPET_METRICS_COLLECTOR_DIR_NAME}"
+    FileUtils.copy_entry src, "#{scale_result_dir}/#{PUPPET_METRICS_COLLECTOR_DIR_NAME}"
+    FileUtils.copy_entry src, "#{scale_results_parent_dir}/#{PUPPET_METRICS_COLLECTOR_DIR_NAME}"
```

Comment:
nitpick, it would be safer to use File.join for these, and since they are both variables now it would be a tiny bit tidier.
```
src = File.join(@archive_root, PUPPET_METRICS_COLLECTOR_DIR_NAME)
FileUtils.copy_entry src, File.join(scale_result_dir, PUPPET_METRICS_COLLECTOR_DIR_NAME)
FileUtils.copy_entry src, File.join(scale_results_parent_dir, PUPPET_METRICS_COLLECTOR_DIR_NAME)
```

Reply:
Nice; updated.